src/components/Nav/LogIn/LogIn.js

```diff
@@ -0,0 +1,101 @@
+import React, { useState } from 'react';
+import { VscGlobe } from 'react-icons/vsc';
+import { VscMenu } from 'react-icons/vsc';
```

Comment:
```suggestion
import { VscGlobe,VscMenu } from 'react-icons/vsc';
```
같은 패키지에서 제공해주는건 이렇게 하나로 합칠 수 있습니다.

Reply:
- [x] 췤